packages/js/generator-grow/generators/phpcs/USAGE

```diff
@@ -0,0 +1,7 @@
+Description:
```

Comment:
💅  WDYT of adding more explanations about the different options???

Reply:
Added in https://github.com/woocommerce/grow/pull/26/commits/00aa70dcb326333a860550cab51eced9c5d77f77.